### PR TITLE
Promote certman operator to production

### PR DESCRIPTION
**Pull request by tparikh**
This commit promotes certman operator version a3d5cd621d6bd698aabd21b30e8265655716a82d to production.

Signed-off-by: Tejas Parikh <tparikh@redhat.com>

### CODE REVIEW

**Comment by tparikh** [comment]
@jharrington22 Please review this PR.

**Review by jharrington22 — APPROVED** [review]
LGTM

**Comment by jharrington22** [comment]
@tparikh @mwoodson @jewzaam this is approved, once merged this will promote certman to prod.

**Pull request merged**
